common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -6,35 +6,54 @@
 
 import com.mojang.blaze3d.platform.Window;
 import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.overlays.overlaySizes.FixedOverlaySize;
+import com.wynntils.core.features.overlays.overlaySizes.OverlaySize;
+import com.wynntils.utils.objects.Pair;
 
 public abstract class Overlay {
-    protected OverlayPosition position;
+    protected OverlayPosition overlayPosition;
```

Comment:
I dunno, since we're in an `Overlay` already, is not `overlayPosition` just longer and redundant? Even if the type is `OverlayPosition` I think the variable can be just `position`. (And the same with overlaySize -> size).